src/python/CRABClient/Commands/submit.py

```diff
@@ -247,6 +248,16 @@ def validateConfig(self):
             msg += " (i.e. will run %s job type)." % ("an analysis" if upper(crab_plugin_name) == 'ANALYSIS' else "a MC generation")
             self.logger.debug(msg)
 
+        ## Check that the requested memory does not exceed the allowed maximum.
+        nCores = getattr(self.configuration.JobType, 'numcores', 1)
```

Comment:
@belforte, is it necessary to specify a default also here (`1`) or is [this](https://github.com/dmwm/CRABClient/blob/a5775c3f10546d15071d5dc79086d84d5d0fe090/src/python/CRABClient/ClientMapping.py#L45) enough?

Reply:
sorry I di not understand what (1) referes to.

Reply:
To the last argument of `getattr`, the only `1` in that line :)

Reply:
let's keep the default for the getattr call.